rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SchedulerRestInterface.java

```diff
@@ -570,7 +570,8 @@ String getJobHtml(
             @QueryParam("pending") @DefaultValue("true") boolean pending,
             @QueryParam("finished") @DefaultValue("true") boolean finished,
             @QueryParam("offset") @DefaultValue("0") int offset,
-            @QueryParam("limit") @DefaultValue("-1") int limit)
+            @QueryParam("limit") @DefaultValue("-1") int limit,
+            @QueryParam("sortparameters") SortSpecifierRestContainer sortParams)
```

Comment:
the other ones i can see in this class are just a single word?Do you mean we have other parameters composed of more than 1 word without camel case?